scripts/lab/remote.sh

```
@@ -0,0 +1,163 @@
+  echo "running sweep on ${#ips[@]} box(es) ..."
+  # The launch runs in a subshell as a setsid/nohup'd background job, so sshd has nothing left to wait for
+  # and ssh returns at once. (A bare `nohup … &` made ssh block until the whole sweep finished, which
+  # serialised the shards.) `timeout` is belt and braces: a hung ssh cannot stall the other launches.
+  i=0
+  for ip in "${ips[@]}"; do
+    $TIMEOUT $SSH@"$ip" "cd /root/openfront && rm -rf /root/lab-out && mkdir -p /root/lab-out && (setsid nohup env CONFIGS='$CONFIGS' MINUTES=$MINUTES SHARD=$i/${#ips[@]} AGGREGATE=0 BATCHES='$batches' ${SPAWNS:+SPAWNS='$SPAWNS'} ${JOBS:+JOBS=$JOBS} ${SHIFT:+SHIFT=$SHIFT} OUT=/root/lab-out bash scripts/lab/sweep.sh > /root/lab-out/sweep.log 2>&1 < /dev/null &); sleep 1; head -1 /root/lab-out/sweep.log" \
+      || echo "WARNING: launch on $ip did not confirm; check /root/lab-out/sweep.log there"
+    i=$((i + 1))
+  done
+  sleep 15
+  running() { for ip in "${ips[@]}"; do $SSH@"$ip" 'pgrep -f "[s]cripts/lab/sweep.sh" >/dev/null' 2>/dev/null && return 0; done; return 1; }
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)